### ISSUE TITLE
Relative file paths / removeable media with changing mountpoints

### ISSUE THREAD

**Issue by kuzmandi** [kzhiquan/Picsee#27]
I would like to suggest that a possibility be created so that the files are stored and found relative to the library.
Now it is the case that the path/mountpoint/volume name to image files must not change.
I would like it if you could also store the files relative to the library file and be udepending on the path/mountpoint/volume name.
This would make handling with removable media much easier.

**Comment by kzhiquan** [comment]
Yes, the idea is good, but for removable media, when mount volume, you have to rebind the linked folders, because of each mac have a unique authorization for the folders. I have write an article to tell how to use Picsee with removable disk, you can view it [in details](https://picsee.chitaner.com/blog/library_in_portable_disk_use.html)

![image](https://user-images.githubusercontent.com/2912180/176910237-7ebb3570-4bc0-4d35-876c-62e99c265cf4.png)

**Comment by kzhiquan** [comment]
@kuzmandi thanks for your suggestion, I am keeping to find a better way to resolve this problem, but until now, as far as I know, it is the best solution.
